Lib/test/test_ssl.py

```diff
@@ -2609,6 +2626,17 @@ def try_protocol_combo(server_protocol, client_protocol, expect_success,
     server_context = ssl.SSLContext(server_protocol)
     server_context.options |= server_options
 
+    min_version = PROTOCOL_TO_TLS_VERSION.get(client_protocol, None)
+    if (min_version is not None
+    # SSLContext.minimum_version is only available on recent OpenSSL
+    # (setter added in OpenSSL 1.1.0, getter added in OpenSSL 1.1.1)
+    and hasattr(server_context, 'minimum_version')
```

Comment:
style nit (not worth another PR, just leave it), I'd have indented these lines to match the column of min_version after your opening ( above.

Reply:
He he, I expected that someone would complain about that 😁 Feel free to change it if you want.